### ISSUE TITLE
Introduction should clarify relationship of Silver to earlier guidelines

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/silver#101]
From https://www.w3.org/2002/09/wbs/94845/Silver-ED-21-01-2020/results#xq3

Make more clear that Silver REPLACES all (WCAG) 2.x (ATAG) 2.0 UAAG 2.0 and should be the ONE AND ONLY reference for now.

For the draft reviewer it remains unclear how ATAG 2.0 and UAAG 2.0 are connected to the current Silver draft and whether silver shall complement or replace these.

**Comment by jspellman** [comment]
done in FPWD
